backend/config/__init__.py

```diff
@@ -43,6 +43,7 @@
 )
 REDIS_HOST: Final = os.environ.get("REDIS_HOST", "localhost")
 REDIS_PORT: Final = os.environ.get("REDIS_PORT", "6379")
+REDIS_PASSWORD: Final = os.environ.get("REDIS_PASSWORD")
```

Comment:
```suggestion
REDIS_PASSWORD: Final = os.environ.get("REDIS_PASSWORD", None)
```

Or maybe it should fallback to `""`? Whichever one will let the no-password option continue to work.

Reply:
By default `os.environ.get()` returns a None if the variable doesn't exists, and the no-password option will work only with None. We can add it as fallback if you want for clearer reading